src/Breadcrumb/Breadcrumb.Component.js

```diff
@@ -1,18 +1,25 @@
+import { Link } from 'react-router-dom';
 import React from 'react';
 import { Breadcrumb, BreadcrumbItem } from '../';
 import { Description, DocsText, DocsTile, Header, Import, Properties, Separator } from '../_playground';
 
 export const BreadcrumbComponent = () => {
     const breadcrumbHrefCode = `<Breadcrumb>
-    <BreadcrumbItem url="#" name="Link Text"/>
-    <BreadcrumbItem url="#" name="Link Text"/>
-    <BreadcrumbItem url="#" name="Link Text"/>
+    <BreadcrumbItem name='Link Text' url='#' />
```

Comment:
rando q: should we have a standard way of handling clicks in the docs rather than `#`?

Reply:
Probably, I think we need to have a pass through all of these docs to improve them. The goal here was just to preserve what was there before while removing router.